### PR TITLE
Update Default airflow version in Helm Chart to 2.6.0

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Review by jedcunningham — APPROVED** [review]

**Review by pankajkoti — APPROVED** [review]

**Comment by pankajkoti** [comment]
Do we use some tool in the Airflow repo to bump the versions across multiple files? I would like to learn more about it. If it's manual, recently, our team at Astronomer was introduced to https://commitizen-tools.github.io/commitizen/bump/ by @Lee-W which helps bump the versions across multiple files with just a `bump` command.

**Comment by potiuk** [comment]
I don't think any tool will do, there are a number of versions of different kinds in different places, but if you can make it happen -feel free to work on it :)

**Comment by potiuk** [comment]
Just to add more context - depends on what version gets upgraded, I usually use IDE support BTW that allows me to review whenever I bump something. And usually there are a number of false positives, and many times there are more things that are needed when you update versions - because we have a number of linked versions in multiple places. And often it happens that automating version upgrade requires some "other" changes next time the verison is bumped, so blind relying on a tool is a bit dangerous (but also manual update is dangerous from another angle).

But we love to automate stuff, and if there is an opportunity - you see - you will find a number of people cheering it.

File: tests/charts/airflow_aux/test_basic_helm_chart.py [hunk 1]
```diff
@@ -37,13 +37,11 @@ def _get_values_with_version(self, values, version):
...
     def _get_object_count(self, version):
-        if version == "2.3.2":
-            return OBJECT_COUNT_IN_BASIC_DEPLOYMENT + 1
-        elif version == "2.6.0":
+        if version == "2.3.2" or version == "default":
             return OBJECT_COUNT_IN_BASIC_DEPLOYMENT + 1
         return OBJECT_COUNT_IN_BASIC_DEPLOYMENT
 
-    @pytest.mark.parametrize("version", ["2.3.2", "2.4.0", "2.6.0", "default"])
+    @pytest.mark.parametrize("version", ["2.3.2", "2.4.0", "default"])
```
Comment (ephraimbuddy) [inline tests/charts/airflow_aux/test_basic_helm_chart.py:44]: 2.6.0 is now the default, so no need to have it and also have a default

**Comment by Lee-W** [comment]
AFAIK, what [commitizen bump](https://commitizen-tools.github.io/commitizen/bump/) can do is replace the old versions in pre-defined places like [version_files](https://github.com/commitizen-tools/commitizen/blob/cbab456d4c96e0f167b274cf1bd18349e4f049e7/pyproject.toml#L4) with new versions (can be manually set to certain version). If we want to fully utilize commitizen auto bumping, we'll need to follow certain commit rule (can be customized). But I'm not sure whether airflow has / is going to have certain commit rules that can be parsed. Nonetheless, changing all the version across pre-definied files might still be something helpful. I'd be interested in working on it if this sounds reasonable.

**Comment by pankajkoti** [comment]
> AFAIK, what [commitizen bump](https://commitizen-tools.github.io/commitizen/bump/) can do is replace the old versions in pre-defined places like [version_files](https://github.com/commitizen-tools/commitizen/blob/cbab456d4c96e0f167b274cf1bd18349e4f049e7/pyproject.toml#L4) with new versions (can be manually set to certain version). If we want to fully utilize commitizen auto bumping, we'll need to follow certain commit rule (can be customized). But I'm not sure whether airflow has / is going to have certain commit rules that can be parsed. Nonetheless, changing all the version across pre-definied files might still be something helpful. I'd be interested in working on it if this sounds reasonable.

@Lee-W could you please also help detail what we mean by certain commit rules so that we can get a better response? I think we can create an issue/discussion and take the conversation there :)

**Comment by Lee-W** [comment]
By default, [commitizen](https://github.com/commitizen-tools/commitizen) uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/). The commit messages look like `type(scope): message` (e.g., `feat(scheduler): cool feature`, `fix(triggerer): hot fix`)

It maps these commits into different bumping. 
圖片

 e.g., `feat(scope): great feature` will be mapped into a `MINOR` bumping.

If a PR contains more than one commit, it uses the one that needs to bump to the highest level.

---

If airflow currently has bumping rules in place, which involve specific commit message patterns (such as "airflow-feature: add a new feature to scheduler"), and version bumping is done based on a mapping (such as `{"airflow-breaking": MAJOR, "airflow-feature": MINOR}`), then automating the bumping process can be taken to the next level.

It basically works like the following.

```python
if "airflow-feature" in commit_msg:
    bump(MINOR)
elif "airflow-fix" in commit_msg:
    bump(PATCH)
elif "airflow-breakchange" in commit_msg:
    bump(MAJOR)
```

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**